src/Tizen.NUI/src/internal/Common/FriendAssembly.cs

```diff
@@ -30,6 +30,7 @@
 
 [assembly: InternalsVisibleTo("Tizen.TV.NUI, " + Tizen.NUI.PublicKey.TizenTV)]
 [assembly: InternalsVisibleTo("Tizen.TV.NUI.Component, " + Tizen.NUI.PublicKey.TizenTV)]
+[assembly: InternalsVisibleTo("Tizen.TV.FLUX.Component.Extension, " + Tizen.NUI.PublicKey.TizenTV)]
```

Comment:
Not an major. Wasnt it looks more beautiful if we define this line under the `Tizen.TV.FLUX.Component`, not the `Tizen.TV.NUI.Component`?

Reply:
Done.